pursuing-toxic-agenda-data.html

```diff
@@ -1,38 +1,34 @@
 ---
 layout: default
-title: "Part 2 - Data Methods"
-description: "Notes on Data Used in Figures"
+section: "2"
+title: "Data Methods"
+subtitle: "Pursuing a Toxic Agenda: Environmental Injustice in the Early Trump Administration"
+description: "Notes on Data Used in Part Figures"
```

Comment:
*"Part X Figures"? Wasn't sure if typo

Reply:
@patcon -- I didn't change any of the content, just the structure in this PR